quinn-proto/src/transport_parameters.rs

```diff
@@ -444,7 +444,13 @@ impl TransportParameters {
                 min_ack_delay.0 > params.max_ack_delay.0 * 1_000
             })
             || (side.is_server()
-                && (params.stateless_reset_token.is_some() || params.preferred_address.is_some()))
+                && (params.original_dst_cid.is_some()
```

Comment:
Should we add a link here to the RFC section you linked in the PR description?

Reply:
Added citations for all checks.